### PR TITLE
Feat: support get_alerts_by_alerts_ids API

### PR DESCRIPTION
**Pull request by guysl10**

### CODE REVIEW

**Review by matany90 — APPROVED** [review]
nice :)

File: intezer_sdk/_api.py [hunk 1]
```diff
@@ -355,14 +356,24 @@ def index_by_file(self, file_path: str, index_as: IndexType, family_name: str =
...
 
+    def get_alerts_by_alert_ids(self, alert_ids: List[str], environments: List[str] = None) -> Tuple[int, List[dict]]:
+        response = self.api.request_with_refresh_expired_access_token(method='GET',
+                                                                      path='/alerts/search',
+                                                                      data=dict(alert_ids=alert_ids,
+                                                                                environments=environments))
+        self._assert_index_response_status_code(response)
+        data_response = response.json()
+
+        return data_response['result']['alerts_count'], data_response['result']['alerts']
```
Comment (davidt99) [inline intezer_sdk/_api.py:375]: It's better to just return the result

**Review by davidt99 — COMMENTED** [review]
Bump version in the `__init__` and add an entry in `CHANGES`

**Review by yoniabrahamy — APPROVED** [review]

File: intezer_sdk/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '1.16.9'
+__version__ = '1.16.10'
```
Comment (davidt99) [inline intezer_sdk/__init__.py:1]: This probably needs to be 1.17

**Pull request merged**